crates/tabby-common/src/config.rs

```diff
@@ -65,6 +86,12 @@ impl RepositoryConfig {
         }
     }
 
+    pub fn validate_name(name: &str) -> bool {
```

Comment:
can we add the regex here and reuse it in webserver? also use regex to validate it here.